### ISSUE TITLE
automated feedback - rust analyzer failed to parse

### ISSUE THREAD

**Issue by geri1701** [exercism/rust-analyzer#38]
The rust analyzer detects a syntax error in my solution, although there is none.
https://exercism.org/tracks/rust/exercises/roman-numerals/solutions/geri1701

**Comment by ErikSchierboom** [comment]
I've looked into this a bit and I think this is due to the `syn` crate that is used to parse the code being outdated. The version that is used is `0.15.30`, whereas the latest version is `1.0.81`. I've tried upgrading but got 20+ compiler errors, most of which will be due to the library using different data types/fields. It would be great if some from @exercism/rust could look into this. 

I'm disabling the analyzer for now until this has been resolved, as it will comment on perfectly valid solutions that they're not actually valid.
